pkg/network/dhcpconfigurator/dhcp_configurator_test.go

```diff
@@ -0,0 +1,77 @@
+package dhcpconfigurator
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/vishvananda/netlink"
+
+	"kubevirt.io/kubevirt/pkg/network/cache"
+	"kubevirt.io/kubevirt/pkg/network/cache/fake"
+)
+
+var _ = Describe("DHCP configurator", func() {
+	var dhcpConfigurationFactory cache.InterfaceCacheFactory
+	var dhcpConfigurator DhcpConfigurator
+
+	const (
+		ifaceName   = "eth0"
+		launcherPID = "self"
+	)
+
+	BeforeEach(func() {
+		dhcpConfigurationFactory = fake.NewFakeInMemoryNetworkCacheFactory()
+	})
+
+	Context("dhcp configurator for bridge interfaces", func() {
```

Comment:
The bridge and the masquerade tests seems the same. What is the difference between them? Shouldn't the `dhcpConfigurator` be agnostic to whether it is `bridge` or `masquerade`?

Reply:
Was it fixed on another commit?

Reply:
Its tricky; in the next commit, what I do is add the `EnsureDHCP` method to the struct, and this one *does* have different behaviors depending on the interface type to plug into the VM.

I guess what I'm trying to say is it can go both ways: just test the import / export for one of the interface types, or test it for both (thus making the unit tests for both interface types similar).